### PR TITLE
return Naive model for constant series in AutoCES

### LINKED ISSUE #811: Getting error when training AutoCES on covid deaths dataset from gluonts
I am trying to fit an AutoCES model to the covid deaths dataset test but I get this exception: `Exception: no model able to be fitted`

I am using the following code 

```python
import pandas as pd
import numpy as np
from gluonts.dataset.repository.datasets import get_dataset
from gluonts.time_feature.seasonality import get_seasonality
from statsforecast.models import AutoCES
from statsforecast import StatsForecast


dataset = get_dataset("covid_deaths", regenerate=False)

# Access the train and test datasets
train_dataset = dataset.train
test_dataset = dataset.test


records = []
for entry in dataset.train:
    target_values = entry["target"]
    item_id = entry["item_id"] if "item_id" in entry else "unknown_item"
    start_date = entry["start"]
    if isinstance(start_date, pd.Period):
        start_date = start_date.to_timestamp()
    # Generate the date range for the target values
    dates = pd.date_range(
        start=start_date, periods=len(target_values), freq=dataset.metadata.freq
    )

    # Append records for each date-target pair along with the item_id
    for target, date in zip(target_values, dates):
        records.append({"y": target, "unique_id": item_id, "ds": date})

# Create a DataFrame from the records
df = pd.DataFrame(records)


season_length = get_seasonality(dataset.metadata.freq)
models = [
    AutoCES(season_length=season_length),
]

model = StatsForecast(
    models=models,
    freq=dataset.metadata.freq,
)

model.fit(df)
```
---

The passed data frame looks like this
Screenshot 2024-04-09 at 5 27 18 PM

And this is the Traceback
```
Traceback (most recent call last):
  File "/Users/moo/Desktop/ReadyTensor/Models/Forecasting/nixtla/statistical_ensemble/test.py", line 47, in <module>
    model.fit(df)
  File "/Users/moo/Desktop/ReadyTensor/Models/Forecasting/nixtla/statistical_ensemble/.venv/lib/python3.11/site-packages/statsforecast/core.py", line 646, in fit
    self.fitted_ = self.ga.fit(
                   ^^^^^^^^^^^^
  File "/Users/moo/Desktop/ReadyTensor/Models/Forecasting/nixtla/statistical_ensemble/.venv/lib/python3.11/site-packages/statsforecast/core.py", line 65, in fit
    raise error
  File "/Users/moo/Desktop/ReadyTensor/Models/Forecasting/nixtla/statistical_ensemble/.venv/lib/python3.11/site-packages/statsforecast/core.py", line 59, in fit
    fm[i, i_model] = new_model.fit(y=y, X=X)
                     ^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/moo/Desktop/ReadyTensor/Models/Forecasting/nixtla/statistical_ensemble/.venv/lib/python3.11/site-packages/statsforecast/models.py", line 963, in fit
    self.model_ = auto_ces(y, m=self.season_length, model=self.model)
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/moo/Desktop/ReadyTensor/Models/Forecasting/nixtla/statistical_ensemble/.venv/lib/python3.11/site-packages/statsforecast/ces.py", line 782, in auto_ces
    raise Exception("no model able to be fitted")
Exception: no model able to be fitted
```

Can someone please help with this error?

Thanks.

### PR DESCRIPTION
**Pull request by jmoralez**
When AutoCES gets constant series it returns an error, this makes it so that it becomes a Naive model instead.

Fixes #811

### CODE REVIEW

File: statsforecast/utils.py [hunk 1]
```diff
@@ -292,6 +292,10 @@ def _ensure_float(x: np.ndarray) -> np.ndarray:
         x = x.astype(np.float32)
     return x
 
+
+def _is_constant(x):
+    return np.all(x[0] == x)
```
Comment (AzulGarza) [inline statsforecast/utils.py:297]: should we use 

https://github.com/Nixtla/statsforecast/blob/23440007294bc2d08c3753a0c84da1c219f0b0e1/statsforecast/arima.py#L1576-L1577 ?

**Review by AzulGarza — CHANGES_REQUESTED** [review]

**Comment by obiii** [comment]
> When AutoCES gets constant series it returns an error, this makes it so that it becomes a Naive model instead.
> 
> Fixes #811

Hi,

I am facing the same problem and in my case, I don't  have any constant columns. Can someone help please?

File: statsforecast/utils.py (hunk 1 above)
Comment (jmoralez) [reply statsforecast/utils.py]: Changed in d222d6c

Comment (AzulGarza) [reply statsforecast/utils.py]: 💙

**Review by AzulGarza — APPROVED** [review]
nice @jmoralez!🎉

**Pull request merged**